src/routes/Home.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Banner from "../components/home-page/Banner";
+import FeaturedBrands from "../components/home-page/FeaturedBrands";
+import FeaturedCategories from "../components/home-page/FeaturedCategories";
+import VideoBanner from "../components/home-page/VideoBanner";
+
+function Home() {
+  return (
+    <div className="relative" style={{ top: "5rem" }}>
```

Comment:
Suggestion, remove inline styles, may be you forgot.

Reply:
No , these are to overwrite the component lib class styles